ios/UARCTModule/UrbanAirshipReactModule.m

```diff
@@ -399,45 +401,34 @@ - (RCTBridge *)bridge {
                  overlay:(BOOL)overlay
                  displayMessage_resolver:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject) {
+    UARCTMessageViewController *mvc = [[UARCTMessageViewController alloc] initWithNibName:@"UAMessageCenterMessageViewController" bundle:[UAMessageCenterResources bundle]];
+    [mvc loadMessageForID:messageId onlyIfChanged:YES onError:nil];
 
-    if (overlay) {
```

Comment:
are we not handling this case anymore? I noticed the displayOverlayMessage method is gone as well.

Reply:
if removing it was intentional, does that mean we should update the api/docs on the js side?

Reply:
removed and updated both platforms and js interface/doc comments